tests/config/test_keys.py

```diff
@@ -38,6 +39,599 @@ def test_keys_for_command_invalid_command(invalid_command: str) -> None:
         keys.keys_for_command(invalid_command)
 
 
+@pytest.mark.parametrize(
+    "valid_command, key_list",
+    [
+        case(
+            "HELP",
+            ["?"],
+        ),
```

Comment:
It's fine to include `case` lines here, though we likely won't need descriptions of each one (via `id`) since they're fairly self-explanatory, which is what we typically use these for.

For now, to make these take up less space, we may want to drop the trailing commas in each case, which will enable black to automatically compress them where possible.

---
These tests are very complete - do they cover every key?

Whether to have tests for every situation (here, key) is difficult, since if they cover specific parameters, then changing the code requires each test to be updated. An alternative can be to consider some common example cases that we want to ensure work, and also decouple the cases from the actual symbol<->key mappings we're using by using mocks.

It's possible that if we adjust the function later, that we won't necessarily need to test this function, but rather the underlying one, which won't have the same degree of coupling to consider.